client/lcd/swagger-ui/swagger.yaml

```diff
@@ -2351,3 +2394,17 @@ definitions:
         type: string
       missed_blocks_counter:
         type: string
+  ParamChange:
+    type: object
+      subspace:
```

Comment:
what if we change this to `module` ? it's cleaner if you're not familiar with the terminology